Source/Scene/ImageryLayer.js

```diff
@@ -793,30 +850,53 @@ define([
         imagery.state = ImageryState.TEXTURE_LOADED;
     };
 
+    function getSamplerKey(minificationFilter, magnificationFilter, maximumAnisotropy) {
+        return minificationFilter + ':' + magnificationFilter + ':' + maximumAnisotropy;
+    }
+
     function finalizeReprojectTexture(imageryLayer, context, imagery, texture) {
-        // Use mipmaps if this texture has power-of-two dimensions.
-        if (!PixelFormat.isCompressedFormat(texture.pixelFormat) && CesiumMath.isPowerOfTwo(texture.width) && CesiumMath.isPowerOfTwo(texture.height)) {
-            var mipmapSampler = context.cache.imageryLayer_mipmapSampler;
+        var minificationFilter = imageryLayer.minificationFilter;
+        var magnificationFilter = imageryLayer.magnificationFilter;
+        var usesLinearTextureFilter = minificationFilter === TextureMinificationFilter.LINEAR && magnificationFilter === TextureMagnificationFilter.LINEAR;
+        // Use mipmaps if this texture uses linear filters and has power-of-two dimensions.
+        if (usesLinearTextureFilter && !PixelFormat.isCompressedFormat(texture.pixelFormat) && CesiumMath.isPowerOfTwo(texture.width) && CesiumMath.isPowerOfTwo(texture.height)) {
+            if (minificationFilter === TextureMinificationFilter.NEAREST) {
```

Comment:
The check for `NEAREST` here is no longer needed.

Reply:
oops